### PR TITLE
2467

### PR DESCRIPTION
**Pull request by RickGove**
Fixed Safari: Narrow table is displayed in my downloadable

### CODE REVIEW

File: scandipwa/.eslintcache [hunk 1]
```diff
@@ -0,0 +1 @@
+[{"/Users/rick/Documents/scandi/EDU-466/EDU-CMA/scandipwa/src/component/MyAccountDownloadable/MyAccountDownloadable.component.js":"1"},{"size":1067,"mtime":1619527250387,"results":"2","hashOfConfig":"3"},{"filePath":"4","messages":"5","errorCount":0,"warningCount":1,"fixableErrorCount":0,"fixableWarningCount":0,"source":null},"1s391xt","/Users/rick/Documents/scandi/EDU-466/EDU-CMA/scandipwa/src/component/MyAccountDownloadable/MyAccountDownloadable.component.js",["6"],{"ruleId":"7","severity":1,"message":"8","line":8,"column":9,"nodeType":"9","messageId":"10","endLine":8,"endColumn":20},"no-console","Unexpected console statement.","MemberExpression","unexpected"]
```
Comment (niklavskatlaps) [inline scandipwa/.eslintcache:1]: Please add this file to the .gitignore

File: scandipwa/src/component/MyAccountMyOrders/MyAccountMyOrders.style.scss [hunk 2]
```diff
@@ -0,0 +1,55 @@
+/**
```
Comment (niklavskatlaps) [inline scandipwa/src/component/MyAccountMyOrders/MyAccountMyOrders.style.scss:1]: Please follow this guide on how to correctly override the core styling of ScandiPWA.
[https://docs.scandipwa.com/stack/override-mechanism/extending-styles](url)

File: scandipwa/src/component/MyAccountMyOrders/MyAccountMyOrders.style.scss [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+    td,
+    th {
+        @include mobile {
+            min-width: 0;
+        }
+    }
+}
+
+.MyDownloadableOrders {
```
Comment (niklavskatlaps) [inline scandipwa/src/component/MyAccountMyOrders/MyAccountMyOrders.style.scss:49]: The styling that you have applied here will be global. Consider moving this to the `MyDownloadableOrders` component or if there isn't such component and these styles should be specific only to the MyAccountMyOrders component, please move them inside of the `.MyAccountMyOrders` selector. 

You can read more about the scss code style structure here: [https://docs.scandipwa.com/structure/scss-code-style](url)

**Review by niklavskatlaps — CHANGES_REQUESTED** [review]

File: scandipwa/src/component/MyAccountMyOrders/MyAccountMyOrders.component.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import 'SourceComponent/MyAccountMyOrders/MyAccountMyOrders.component';
```
Comment (niklavskatlaps) [inline scandipwa/src/component/MyAccountMyOrders/MyAccountMyOrders.component.js:1]: I would say that that this file should look something like this:
```
import { MyAccountMyOrders } from 'SourceComponent/MyAccountMyOrders/MyAccountMyOrders.component';

import './MyAccountMyOrders.override.style';

export default MyAccountMyOrders;
```

Comment (niklavskatlaps) [reply scandipwa/src/component/MyAccountMyOrders/MyAccountMyOrders.component.js:1]: Oh, I see, you copied everything from the ScandiPWA documentation - let's keep it the way it is then! 

Just a few notes:
- You can import styles without the .scss extension, too. Not sure which one is the preferred approach right now.
- I am not sure whether that `export * from '...'` is needed in your case since there are no other exports in the Source component but I guess it won't hurt if you keep it the way it is.

Bottom line is that this file could have looked different and it would be alright in both cases. I guess the preferred approach would be the one that ScandiPWA CLI tools would generate and/or the one that we have in the docs.

**Review by niklavskatlaps — APPROVED** [review]

**Review by niklavskatlaps — APPROVED** [review]
